mittab/libs/tab_logic.py

```diff
@@ -619,13 +638,13 @@ def avg_deb_speaks(debater):
         return float(sum(real_speaks)) / float(len(real_speaks))
 
 
-def debater_forfeit_speaks(debater):
-    """ Calculate a debater's speaks for a forfeit round
+def debater_forfeit_speaks(debater, round_number):
+    #Calculate a debater's speaks for a forfeit round
 
-    Note that right now we just return 0, but we may want to add support
-    for returning average speaks or some such
-    """
-    return 0.0
+    if (TabSettings.get('lenient_late') >= round_number):
```

Comment:
So I'm thinking we either stick with this or alter the NoShow table to have an additional flag. I'm sorta ok with their being a default "lenient" round that we can change via TabSettings as long as the default is 0.

I think the big issue I've always had with lenient noshows is that having a hardline policy of "if you're late you're tabbed at 0" has been good for making people actually show up on time, but maybe I'm out of touch on that. I'm fine with it as an option anyhow.